PSUtil/functions/path/Set-PSUPathAlias.ps1

```diff
@@ -0,0 +1,63 @@
+﻿function Set-PSUPathAlias {
+    <#
+	.SYNOPSIS
+		Used to create an an alias that sets your location to the path you specify.
+
+	.DESCRIPTION
+		A detailed description of the Set-PSUPathAlias function.
+
+	.PARAMETER Alias
+		Name of the Alias that will be created for Set-PSUPath.
+		Set-PSU Path detects the alias that called it and then find the corresponding PSFConfig entry for it.
+
+	.PARAMETER Path
+		This is the path that you want your location to change to when the alias is called.
+
+	.PARAMETER Register
+        Registers the configuration setting to the UserDefault scope. This allows you to set aliases quickly.
+        For more advanced options, see Register-PSFConfig.
```

Comment:
This has little meaning to beginner users.
Maybe add a line first about it causing PSUtil to remember the alias across sessions